presto-common/src/main/java/com/facebook/presto/common/block/Block.java

```diff
@@ -164,7 +164,7 @@ default int compareTo(int leftPosition, int leftOffset, int leftLength, Block ri
 
     /**
      * Gets the value at the specified position as a single element block.  The method
-     * must copy the data into a new block.
+     * must copy the data into a new block, if there are more elements in the block.
```

Comment:
I'd rephrase it with "more than one element", something like this:
`if there are more elements in the block` -> `if there are more than one element in the block`